tracer/src/Datadog.Trace/Iast/Aspects/System.Text/StringBuilderAspects.cs

```diff
@@ -58,9 +63,9 @@ public static StringBuilder Init(string value, int startIndex, int length, int c
     /// <param name="target"> StringBuilder instance </param>
     /// <returns> instance.ToString() </returns>
     [AspectMethodReplace("System.Object::ToString()", "System.Text.StringBuilder")]
-    public static string ToString(StringBuilder target)
+    public static string ToString(StringBuilder? target)
     {
-        var result = target.ToString();
+        var result = target!.ToString();
```

Comment:
If you need to add `?` to the signature, to say the parameter can be null, then adding a `!` target to say it can neve be null seems wrong.

Reply:
Or you want to trigger a null ref here for compatibility? If so, please add a comment to say so, as you do below.

Reply:
Yes, the idea is to force the aspect to launch an exception when calling the original method when null. I will add a comment as in the other places.

Reply:
Comment addded!

Reply:
IMHO I think @nullable enable in this kind of files is counter productive. Although method signature is not affected by the nullability "?" of the parameter (luckily), we are adding a layer of semantic that is not present in the rewritten methods, and it tangles the code.